pkgs/applications/networking/irc/chatterino2/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, pkgconfig, fetchgit, qt5, boost, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "chatterino2";
+  version = "2.0.4";
+  name = "${pname}-${version}";
+  src = fetchgit {
+    url = "https://github.com/fourtf/${pname}.git";
```

Comment:
You can use `fetchFromGitHub` instead of `fetchgit` here.

Reply:
Oh, ok! After a brief googling I came to a conclusion that `fetchFromGitHub` doesn't support submodules. Just double checked the source code and noticed `fetchSubmodules`. I'll try it now. Thanks!